Buildings/Applications/DataCenters/ChillerCooled/Examples/IntegratedPrimaryLoadSideEconomizer.mo

```diff
@@ -2,27 +2,40 @@ within Buildings.Applications.DataCenters.ChillerCooled.Examples;
 model IntegratedPrimaryLoadSideEconomizer
   "Example that demonstrates a chiller plant with integrated primary load side economizer"
   extends Modelica.Icons.Example;
-  extends
-    Buildings.Applications.DataCenters.ChillerCooled.Examples.BaseClasses.PartialDataCenter(
+  extends Buildings.Applications.DataCenters.ChillerCooled.Examples.BaseClasses.PostProcess(
+    freCooSig(
+      y=if cooModCon.y == integer(Buildings.Applications.DataCenters.Types.CoolingModes.FreeCooling)
+      then 1 else 0),
+    parMecCooSig(
+      y=if cooModCon.y == integer(Buildings.Applications.DataCenters.Types.CoolingModes.PartialMechanical)
+      then 1 else 0),
+    fulMecCooSig(
+      y=if cooModCon.y == integer(Buildings.Applications.DataCenters.Types.CoolingModes.FullMechanical)
+      then 1 else 0),
+    PHVAC(y=cooTow[1].PFan + cooTow[2].PFan + pumCW[1].P + pumCW[2].P + sum(
+          chiWSE.powChi + chiWSE.powPum) + ahu.PFan + ahu.PHea),
+    PIT(y=roo.QSou.Q_flow));
+  extends Buildings.Applications.DataCenters.ChillerCooled.Examples.BaseClasses.PartialDataCenter(
     redeclare Buildings.Applications.DataCenters.ChillerCooled.Equipment.IntegratedPrimaryLoadSide chiWSE(
       addPowerToMedium=false,
       perPum=perPumPri,
       tauPump=1,
       use_controller=false,
-      use_inputFilter=true,
-      energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial),
+      energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial,
+      riseTimePump=30),
```

Comment:
30 seconds rise time for pumps is the default in the library. Why is it set to 120 seconds in `IntegratedPrimaryLoadSide`? Also, `IntegratedPrimaryLoadSide` sets `tauPump=30` while the library uses `1` second. Why is this overwritten to another value? Unless there is a reason, we should be consistent.